extra/resources/HealthCPU

```diff
@@ -33,7 +33,7 @@ meta_data() {
     cat <<END
 <?xml version="1.0"?>
 <resource-agent name="HealthCPU" version="1.0">
-<version>1.0</version>
+<version>1.1</version>
```

Comment:
We can update the resource-agent version as well, since we're changing it a good bit